src/components/History.tsx

```diff
@@ -6,6 +6,13 @@ const History = (props: Props): JSX.Element => {
   return (
     <table className="govuk-table lbh-table">
       <tbody className="govuk-table__body">
+        <tr className="govuk-table__row">
+          <td className="govuk-table__cell"></td>
+          <td className="govuk-table__cell">Valid until</td>
+          <td className="govuk-table__cell">
+            {props.documentSubmission.validUntil.toLocal().toFormat('d LLLL y')}
```

Comment:
You should be able to apply `formatDate` to `validUntil`?

Reply:
👍